rb1_02.rb

```diff
@@ -0,0 +1,4 @@
+ary = Array[1, 2, 3, 5, 6, 3, 5]
```

Comment:
Can we directly use `[1, 2, 3, 5, 6, 3, 5]`?
```suggestion
ary = [1, 2, 3, 5, 6, 3, 5]
```